docs/config.md

```diff
@@ -1,8 +1,13 @@
 # Configuration
 
-You can configure the app by copying `config.sample.json` to `config.json` and customising it. The possible options are
-described here. If you run into issues, please visit [#element-web:matrix.org](https://matrix.to/#/#element-web:matrix.org)
-on Matrix.
+You can configure the app by copying `config.sample.json` to `config.json` or `config.$domain.json` and customising it.
+Element will attempt to load first `config.$domain.json` and if it fails `config.json`. This mechanism allows different
+configuration options depending on if you're hitting e.g. `app1.example.com` or `app2.example.com`. Configs are not mixed
+in any way, it either entirely uses the domain config, or entirely uses `config.json`. Using `config.json` is the
+recommended default solution.
```

Comment:
What leads you to think it's recommended?

Reply:
I don't have a strong preference to this.
The thought was that if it wasn't, an alternative would have been documented over the last 4 years.
Public element.io seems to use it that way too.

Reply:
I would agree that it arguably is the "standard" way to do this though I wouldn't go as far as to say it is recommended. My suggestion here would be just to remove the sentence as it feels unnecessary

Reply:
I have removed recommendation.